NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -90,17 +90,28 @@ public extension NeuroID {
         NeuroID.saveEventToLocalDataStore(nidEvent)
     }
     
+    internal static func captureLOGEvent(_ message: String, level: String) {
+        let logEvent = NIDEvent(type: .log)
+        logEvent.m = message
+        logEvent.level = level
+        
+        NeuroID.saveEventToLocalDataStore(logEvent)
+    }
+    
     /**
      Based on the parameter passed in AND the sampling flag, this function will make a call to the ADV library or not,
      Default is to use the global settings from the NeuroID class but can be overridden (see `start`
      or `startSession` in the `NIDAdvancedDevice.swift` file.
      
      Marked as `@objc` because this method can be called with reflection if the ADV library is not installed.
-     Because of the reflection we use an array with a boolean instead of just boolean
+     Because of the reflection we use an array with a boolean instead of just boolean. Log the shouldCapture flag
+     in a LOG event (isAdvancedDevice setting: <true/false>.
      */
     @objc internal static func captureAdvancedDevice(
         _ shouldCapture: [Bool] = [NeuroID.isAdvancedDevice]
     ) {
+        captureLOGEvent("isAdvancedDevice setting: \(shouldCapture)", level: "INFO")
```

Comment:
NIT: This is the `shouldCapture` setting not `isAdvancedDevice`. Speaking of which do we/can we capture a log event on the `configure` method that indicates what the `isAdvancedDevice` setting was passed in as?